src/components/organisms/AdminVenueView/components/AdminSidebarFooter/AdminSidebarFooter.scss

```diff
@@ -0,0 +1,48 @@
+@import "scss/constants.scss";
+
+.AdminSidebarFooter {
+  position: absolute;
+  bottom: $footer-height;
+
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+  background-color: $black;
+  border-top: 1px solid opaque-white(0.2);
+
+  padding: $spacing--xs;
+
+  width: $admin-ng-sidebar-width--normal;
+  min-width: $admin-ng-sidebar-width--min;
+  max-width: $admin-ng-sidebar-width--max;
+  max-height: $admin-ng-sidebar-footer-height--max;
+
+  font-size: 1rem;
+  line-height: 1rem;
```

Comment:
Why do we need to define font-related styles in `AdminSidebarFooter`. It has only buttons with already defined fonts styles

Reply:
`rem` is relative to page while `em` or whatever inherited from parent might be really off. This ensures the footer has consistent font size and line hight no matter where it is put.